Cannot find mongoose module with webpack

Getting the following error with webpack and mongoose - `Uncaught Error: Cannot find module '././binary_parser'`

Below is my webpack.config.js and the mongoose version I am using is 4.3.3
```
var path = require('path');
var webpack = require('webpack');
var config = require('./tasks/config');

module.exports = {
  context: path.join(__dirname, 'src'),
  devtool: 'eval',
  historyApiFallback: true,
  entry: {
    app: [
      'webpack-hot-middleware/client',
      './' + config.src.main
    ]
  },
  target: 'node',
  output: {
    path: path.join(__dirname, config.dest),
    filename: 'bundle.js',
    publicPath: '/static/'
  },
  node: {
    fs: 'empty',
    net: 'empty',
    tls: 'empty'
  },
  plugins: [
    new webpack.DefinePlugin({
      'process.env': {
        NODE_ENV: JSON.stringify(process.env.NODE_ENV || 'development')
      }
    }),
    new webpack.HotModuleReplacementPlugin(),
    new webpack.IgnorePlugin(/vertx/),
    new webpack.NoErrorsPlugin()
  ],
  resolve: {
    alias: {
      'redux': path.join(__dirname, 'node_modules/redux')
    },
    extensions: ['', '.js', '.json', '.node'],
    fallback: path.join(__dirname, 'node_modules')
  },
  resolveLoader: { fallback: path.join(__dirname, 'node_modules') },
  module: {
    loaders: [{
      test: /\.js$/,
      loaders: ['react-hot', 'babel'],
      exclude: /node_modules/
    }, {
      test: /\.scss$/,
      loaders: ['style', 'css', 'sass'],
      include: __dirname
    }, {
      test: /\.html$/,
      exclude: /node_modules/,
      loader: 'file?name=[path][name].[ext]'
    }, {
      test: /\.json$/,
      loader: 'json'
    }, {
      test: /\.node$/,
      loader: 'node'
    }]
  }
};
```

The error is being thrown here 
```
function webpackContextResolve(req) {
	return map[req] || (function() { throw new Error("Cannot find module '" + req + "'.") }());
};
```

Can someone please point me in the right direction????

--- a/lib/document_provider.web.js
+++ b/lib/document_provider.web.js
@@ -1,0 +1,17 @@
+'use strict';
+
+/* eslint-env browser */
+
+/*!
+ * Module dependencies.
+ */
+var BrowserDocument = require('./browserDocument.js');
+
+/**
+ * Returns the Document constructor for the current context
+ *
+ * @api private
+ */
+module.exports = function() {
+  return BrowserDocument;
+};

--- a/lib/drivers/index.web.js
+++ b/lib/drivers/index.web.js
@@ -1,0 +1,1 @@
+module.exports = require('./browser');

